src/marisa_trie.pyx

```diff
@@ -248,33 +254,37 @@ cdef class _Trie:
         self._trie.mmap(c_path)
         return self
 
-    def iterkeys(self, unicode prefix=""):
+    def iterkeys(self, prefix=NoPrefixGiven):
```

Comment:
I'm not sure I understand the reason for a custom singleton here. Could you please elaborate on why wouldn't `""` or '`None` work?